lock-api-objects/src/main/java/com/palantir/lock/watch/NoOpLockWatchEventCache.java

```diff
@@ -55,10 +52,19 @@ public CommitUpdate getCommitUpdate(long startTs) {
     @Override
     public TransactionsLockWatchEvents getEventsForTransactions(Set<Long> startTimestamps,
             Optional<IdentifiedVersion> version) {
-        return NONE;
+        IdentifiedVersion fakeVersion = generateFakeVersion();
+        return ImmutableTransactionsLockWatchEvents.builder()
+                .clearCache(true)
+                .startTsToSequence(
+                        startTimestamps.stream().collect(Collectors.toMap(startTs -> startTs, $ -> fakeVersion)))
+                .build();
     }
 
     @Override
     public void removeTransactionStateFromCache(long startTimestamp) {
     }
+
+    private IdentifiedVersion generateFakeVersion() {
```

Comment:
nit: If we want to keep this, this should be static. Though I think having a constant for the UUID would be preferable

Reply:
As per the other, I agree.